### ISSUE TITLE
[accessibility] Disabled tabs are keyboard operable

### ISSUE THREAD

**Issue by Malvoz** [Turbo87/sidebar-v2#237]
Disable tabs should not be activatable using a keyboard, and the state should be `aria-disabled="true"`. Or they could be set to `tabindex="-1"`, taking them out of the tab order.
